npc/re/quests/illusion_investigation.txt

```diff
@@ -126,33 +126,33 @@ prt_in,136,34,3	script	Leader Joel#joel	1_M_ORIENT01,{
 	close;
 
 OnInit:
-	questinfo( QTYPE_EVENT, QMARK_YELLOW, "isbegin_quest(3470) == 0" );
+	questinfo( QTYPE_QUEST, QMARK_YELLOW, "isbegin_quest(3470) == 0" );
 	end;
 }
 
 prt_in,136,29,3	script	Unit Zenhi#zenhi	4_F_KHELLY,{
```

Comment:
"Unit" should be "Member". The "Unit" translations is only used in books (ex. Page 3, Unit 4)